x-pack/filebeat/module/salesforce/_meta/docs.asciidoc

```diff
@@ -0,0 +1,303 @@
+[role="xpack"]
+
+:modulename: salesforce
+:has-dashboards: false
+
+== Salesforce module
+
+This is a module for Salesforce logs. 
+It collects the logs using the following two different types of inputs:
+
+- `httpjson` input: collects historical data from Salesforce REST API.
+- `cometd` input: collects real-time data from Salesforce Streaming API.
+
+It includes the following filesets for receiving logs:
+
+- `login-rest` fileset: supports Salesforce Login logs received from the REST API.
+- `login-stream` fileset: supports Salesforce Login logs received from the Streaming API.
+- `logout-rest` fileset: supports Salesforce Logout logs received from the REST API.
+- `logout-stream` fileset: supports Salesforce Logout logs received from the Streaming API.
+- `apex-rest` fileset: supports Salesforce Apex logs received from the REST API.
+- `setupaudittrail-rest` fileset: supports logs generated when admins make in your org’s Setup area.
+

```

Comment:
Would it be helpful to explain somewhere that for the rest of the events, users could use inputs provided above?

Reply:
Makes sense, let me update that. Thanks!